data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -1440,8 +1440,8 @@ spec:
                 description: MachineNetwork is the list of IP address pools for machines.
                   This field replaces MachineCIDR, and if set MachineCIDR must be
                   empty or match the first entry in the list. Default is 10.0.0.0/16
-                  for all platforms other than libvirt. For libvirt, the default is
-                  192.168.126.0/24.
+                  for all platforms other than libvirt and powervs. For libvirt, the
```

Comment:
Unless as variable name, always use `Power VS`. Here you have a mix of `powervs` and `Powervs.`